src/test/java/guitests/SaveCommandTest.java

```diff
@@ -0,0 +1,61 @@
+package guitests;
+
+import java.io.File;
+import java.util.Random;
+
+import org.junit.Test;
+
+import seedu.multitasky.logic.commands.SaveCommand;
+import seedu.multitasky.testutil.TestUtil;
+
+//@@author A0132788U
+/**
+ * Tests for Save Command. Include valid path tests, non XML file, existing file, non-writable file and invalid parent.
+ */
+public class SaveCommandTest extends EntryBookGuiTest {
+    /***************************
+     * Set to a valid file path *
+     **************************/
+    @Test
+    public void saveToValidPath_success() {
+        String validPath = TestUtil.getFilePathInSandboxFolder(Integer.toString(new Random().nextInt(250000)) + ".xml");
+        new File(validPath).getParentFile().setWritable(true);
+        commandBox.runCommand("save " + validPath);
+        assertResultMessage(String.format(SaveCommand.MESSAGE_SUCCESS + validPath));
+    }
+
+    /***************************
+     * End file path with non XML extension *
+     **************************/
+    @Test
+    public void saveFilePathToNonXml_failure() {
+        String nonXmlFilePath = TestUtil.getFilePathInSandboxFolder("entrybook.fxml");
+        commandBox.runCommand("save " + nonXmlFilePath);
+        assertResultMessage(String.format(SaveCommand.MESSAGE_FAILURE + SaveCommand.MESSAGE_USAGE));
```

Comment:
for the failures, will need to do a thrown.expect(CommandException.class) 
and a thrown.expectMessage(MESSAGE_FAILURE)
i think i've got a few examples from edit or delete tests to reference from.